src/offices/_single.html

```diff
@@ -91,7 +91,7 @@ <h1 class="h3">{{ top_story.head | safe }}</h1>
                     block__padded-top
                     block__border-top">
         <h1 class="h2">
-            {{ resource_title }}
+           {{ resource_title }}
```

Comment:
Oh, hmm, maybe to give the section a title for screenreaders hardcode a value like this:

```
<h2 class="u-visually-hidden">
Resources
</h2>
```

Otherwise, I say remove it completely.